pkg/cmd/admin/prune/builds.go

```diff
@@ -97,7 +97,7 @@ func NewCmdPruneBuilds(f *clientcmd.Factory, parentName, name string, out io.Wri
 					return nil
 				}
 			default:
-				fmt.Fprintln(os.Stderr, "Dry run enabled - no modifications will be made.")
+				fmt.Fprintln(os.Stderr, "Dry run enabled - no modifications will be made. add --confirm to remove images")
```

Comment:
This is the prune builds subcommand. Can you capitalize Add, correct this to say builds, and update the deployments and images subcommands as well?